src/public/utils.ts

```diff
@@ -38,7 +38,13 @@ function stopWaiting() {
  * Go back to linkurious home page
  */
 function goToLinkurious(sourceKey: string) {
-  const location = window.location.origin + `/dashboard?key=${sourceKey}`;
+  let basePath = window.location.pathname.split('/')[1];
```

Comment:
if you do that `basePath` will never be undefined because:
- when no base folder is configured then pathname is `/plugins/importer' and `basePath` is `plugins`
- when a base folder is configured then pathname is `/:baseFolder/plugins/importer' and `basePath` is `:baseFolder`

Instead, I suggest to do:
```
const match = window.location.pathname.match(/([^/]+)\/plugins\/.+/)
if match is null then baseBath is ''
else basePath is match[1]
```